e2e/test/framework/framework.go

```diff
@@ -0,0 +1,65 @@
+package framework
+
+import (
+	"github.com/appscode/go/crypto/rand"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+var (
+	Image    = "linode/linode-cloud-controller-manager:latest"
+	ApiToken = ""
+
+	KubeConfigFile         = ""
+	testServerResourceName = "e2e-test-server-" + rand.Characters(5)
```

Comment:
I think we should change this name.

If it will be used for resource types other than services, then the prefix should be "e2e-test-".

If it will only be used for services, then it should be defined in service.go, and the prefix should be "e2e-test-service-"